src/Templates/src/templates/maui-mobile/Pages/ProjectDetailPage.xaml

```diff
@@ -118,20 +118,19 @@
                             <Border StrokeShape="RoundRectangle 22" 
                                 HeightRequest="44" 
                                 VerticalOptions="Center"
-                                BackgroundColor="Transparent"
                                 Padding="{OnPlatform '18,0,18,8',Android='18,0,18,0'}"
                                 SemanticProperties.Description="{Binding Title}">
                                 <VisualStateManager.VisualStateGroups>
                                     <VisualStateGroup x:Name="CommonStates">
                                         <VisualState x:Name="Normal">
                                             <VisualState.Setters>
-                                                <Setter Property="Background" Value="{AppThemeBinding Light={StaticResource LightSecondaryBackground},Dark={StaticResource DarkSecondaryBackground}}"/>
+                                                <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource LightSecondaryBackground},Dark={StaticResource DarkSecondaryBackground}}"/>
```

Comment:
Requires because `LightSecondaryBackground` and `DarkSecondaryBackground` resources are Colors?

Reply:
- ProjectDetailPage.xaml: Uses BackgroundColor (fix for overflow)
- ProjectListPage.xaml: Uses Background (added explicitly)